allow astropy 7.x (#84)
echemdb/unitpackage: doc/news/astropy.rst

**Changed:**

> * Changed maximum astropy version to 7.x.

Reviewer: ```suggestion
* Changed upper version bound for astropy from <=7 to <=8.
```

Reviewer: Maybe we should change it to `<9` to be consistent with how we indicate all other packages.

> * Changed ubuntu version for build doc to 24.04.

Reviewer: ```suggestion
* Changed Ubuntu version for building the documentation in GitHub workflows to 24.04.
```

Reviewer: Do we indicate changes in the repository that are not relevant to the functionality of the module?

Author: Probably not.

> * Changed ubuntu version for build doc to 24.04.

Author: ```suggestion
```
